Source/WebCore/platform/graphics/cg/ImageBufferCGBitmapBackend.cpp

```diff
@@ -54,28 +54,34 @@ std::unique_ptr<ImageBufferCGBitmapBackend> ImageBufferCGBitmapBackend::create(c
     if (backendSize.isEmpty())
         return nullptr;
 
-    void* data;
-    unsigned bytesPerRow = 4 * backendSize.width();
+    CheckedSize bytesPerRow = checkedProduct(4, backendSize.width());
+    if (bytesPerRow.hasOverflowed())
+        return nullptr;
 
-    if (!tryFastCalloc(backendSize.height(), bytesPerRow).getValue(data))
+    CheckedSize numBytes = backendSize.height();
+    numBytes *= bytesPerRow;
```

Comment:
Not sure why the case above uses `checkedProduct` and this case uses `*=` but both seem OK

Reply:
Ok, I'll make it consistent.